variables.tf

```diff
@@ -84,3 +84,18 @@ variable "deletion_protection" {
   description = "Set deletion protection on the instance"
   default     = true
 }
+
+variable "monitoring_interval" {
+  description = "The interval, in seconds, between points when Enhanced Monitoring metrics are collected for the DB instance."
+  default     = 30
+}
+
+variable "performance_insights_enabled" {
+  description = "Specifies whether Performance Insights are enabled"
+  default     = true
```

Comment:
What are the cost implications of enabling performance insights on 74 RDS instances?

I think we probably want to default this to false, and enable it as needed.

Reply:
Yeah it will be high, Thanks. changed it to false by default.